BakuBakuChan/performance_testing_mentoring_program_2020
# [PK] Merge first task to master branch and mark it as "Done"

Task 1. _Simple script_
_Prerequisites:_

1. Manually test site’s workability

_Description:_

1. Create simple script to open Home page, open Contacts.
2. Number of users: up to 10
3. User role: anonymous

_Goals:_

1. Check availability of the site from the load tool
2. Make sure the site is configured for multiuser mode
3. Setup basic metrics and make sure they can be collected
4. Setup ramp-up mode in the scenario
